src/databases/Adventure/CMakeLists.txt

```diff
@@ -1,66 +1,13 @@
 # DO NOT EDIT THIS FILE! THIS FILE IS AUTOMATICALLY GENERATED BY xml2cmake
```

Comment:
I have a question - does xml2cmake make the changes you have included in this PR? Or did you make the changes manually?

Reply:
The changes were added to GenerateCMake.h (xml2cmake).